essential-projects/iam_browser
# 🔥 Remove jQuery and jQuery Cookie Usage

## What did you change?

When building the schemas, i got this error:

```
> gulp typescript-schema

[09:36:44] Using gulpfile ~/Desktop/Projekte/BPMN/process_engine_meta/iam_browser/gulpfile.js
[09:36:44] Starting 'typescript-schema'...

/Users/heiko/Desktop/Projekte/BPMN/process_engine_meta/iam_browser/src/iam_service.ts (62,32): Property 'cookie' does not exist on type 'JQueryStatic<HTMLElement>'.
errors during TypeScript compilation - exiting...
```

The cookie part is no longer needed, so its removed.

## How can others test the changes?

- Try `npm run build-schemas` without this fix
- See how you get the error i mentioned
- Try `npm run build-schemas` with this fix
- See how it now works

## PR-Checklist

Please check the boxes in this list after submitting your PR:
